### ISSUE TITLE
Cannot update solution-only packages

### ISSUE THREAD

**Issue by o-o00o-o** [NuGet/Home#1272]
I have a nuget package I've built that has a few files in custom directories. This should and does behaviour as a solution-only package in that it creates a .nuget folder in the solution and maintains a packages.config in there.

I've then updated that nuget package but it seems there is no way for me to be able to update it.

Using VS "Manage Nuget Packages for Solution". It see's the new version and can see that it is available but it is not selectable. 

![vs](https://cloud.githubusercontent.com/assets/13693159/9528595/451f53b8-4ced-11e5-9d2b-0672bcddc2f3.png)

Equally if I try to use the command line tool nuget update .\mysolution.sln it comes back 

Scanning for projects...
No projects found with packages.config.

What is the point of solution-only packages if we can't update them?

**Comment by emgarten** [comment]
This looks like a scenario that should work. Would you give us more information on the package (I don't see it on nuget.org) and your solution. Is the package installed into both the solution and a project?

**Comment by o-o00o-o** [comment]
Sorry I don't, I since ended up making it a content package with a surrogate project which actually worked better for my scenario as the support files were always in the same place regardless of the version.

It was a very simple package with all files in custom folders (none of the standard ones content/tools etc) no scripts or anything clever and only content files so would be easy to reproduce. The package may have been installed into both a solution and a project - I was playing around with different approaches and was new to nuget so possible I caused an unusual problem

As I see now that solution-only packages are also on the way out of support and I've now found a workaround and if you believe that it will work, happy for this to be closed.
